backend/utils/volunteer-sheet-parser.js

```diff
@@ -0,0 +1,87 @@
+// run `node utils/volunteer-sheet-parser.js <insert_spreadsheet_path>` from the `/backend` folder
+
+const XLSX = require("xlsx");
+const extractor = require("keyword-extractor");
+const resourceUtils = require("./resource-utils");
+const Resource = require("../models/Resource");
+const mongoose = require("mongoose");
+
+mongoose.connect(process.env.DB_URI, {
+  useUnifiedTopology: true,
+  useNewUrlParser: true
+});
+
+const formatNotes = ({
+  "Form Received": formReceived,
+  Availability: availability,
+  "How did you hear about LAH?": refer,
+  "Volunteer Roles": roles
+}) =>
+  `Form Received: ${ 
+  formReceived 
+  }\n\n` +
+  `Availability: ${ 
+  availability 
+  }\n\n` +
+  `How they heard about LAH: ${ 
+  refer 
+  }\n\n` +
+  `Volunteer Roles: ${ 
+  roles 
+  }\n\n`;
+
+const createTags = ({
+  "18 or Older?": eighteen,
+  "Skills, Qualifications, Current Occupation": skills,
+  "Do you have a degree in the mental health field?": mentalHealth,
+  "Volunteer Roles": roles,
+  "Willing to travel?": travel
+}) => [
+  eighteen === "Yes" ? "18+" : null,
+  mentalHealth === "Yes" ? "Mental Health" : null,
+  travel === "Yes" ? "Able to travel" : null,
+  ...extractor.extract(`${skills } ${ roles}`, {
+    language: "english",
+    remove_digits: true,
+    return_changed_case: true,
+    remove_duplicates: true
+  })
+];
+
+const getLocation = async address => {
+  try {
+    const { lat, lng } = await resourceUtils.addressToLatLong(address);
+    return [lng, lat];
+  } catch (err) {
+    return [null, null];
+  }
+};
+
+const convertSchema = async entry => ({
+  companyName: "a",
+  contactName: `${entry["First Name"] } ${ entry["Last Name"]}`,
+  contactPhone: "a",
+  contactEmail: entry["Email Address"] || "a",
+  description: entry["Skills, Qualifications, Current Occupation"] || "a",
+  address: entry["Mailing Address"] || "a",

```

Comment:
What does `"a"` symbolize?

Reply:
LOL its a filler for now since those entries are mocked out in the example spreadsheet, and the mongo insert will fail if they are the empty string. This shouldn't be merged with the "a"s, I'll remove them beforehand. I think I'm waiting on some schema changes I'm supposed to work on before merging this anyways.